pkg/project/resource_project_repo_test.go

```diff
@@ -114,3 +116,131 @@ func TestAccProjectRepo(t *testing.T) {
 		},
 	})
 }
+
+/*
+Test to assign large number of repositories to a project
+*/
+func TestAccAssignMultipleReposInProject(t *testing.T) {
+
+	const numRepos = 100
+	const repoNameInitial = "repo-"
+
+	name := "tftestprojects" + randSeq(10)
+	resourceName := "project." + name
+	projectKey := strings.ToLower(randSeq(6))
+
+	randomRepoNames := func(repoCount int) []string {
+		var repoNames []string
+		for i := 0; i < repoCount; i++ {
+			repoNames = append(repoNames, fmt.Sprintf("%s%s", repoNameInitial, randSeq(10)))
+		}
+		return repoNames
+	}(numRepos)
+
+	repoNamesStr := func(repoNames []string) string {
+		jsonStr, err := json.Marshal(repoNames)
+		if err != nil {
+			return "[]"
+		}
+		return string(jsonStr)
```

Comment:
Why the need to convert to string again?

Reply:
jsonStr is of type []byte

Reply:
renamed variable